desktop/src/com/mygdx/game/client/input/SubFieldUiInputProcessor.java

```diff
@@ -0,0 +1,28 @@
+package com.mygdx.game.client.input;
+
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.InputAdapter;
+import com.badlogic.gdx.utils.viewport.Viewport;
+import com.mygdx.game.client.GdxGame;
+import lombok.NonNull;
+
+public class SubFieldUiInputProcessor extends InputAdapter {
+
+  private final GdxGame game;
+
+  public SubFieldUiInputProcessor(@NonNull GdxGame game) {
+    this.game = game;
+  }
+
+  @Override
+  public boolean keyDown(int keycode) {
+    switch (keycode) {
+      case Input.Keys.ESCAPE -> backToGameScreen();
+    }
+    return true;
+  }
+
+  private void backToGameScreen() {
+    game.changeToGameScreen();
```

Comment:
Czy usuwasz tego inputprocessora jak wychodzisz ze screena subfieldów? bo on może być cały czas aktywny

Reply:
Myślałem, że inputprocessory same się dezaktywują przy zmianie screena, jak się usuwa input processory?

Reply:
Sprawdziłem i nie jest aktywny

Reply:
chyba trzeba nullować, ale możesz to też przetestować z jakimś logiem czy tak faktycznie jest btw

Reply:
to super xD sorry, że nie pamiętałem